### DISCUSSION TITLE
How to make wagtail CharBlock not translatable (wagtail_localize)

### DISCUSSION THREAD

**Discussion by 2567910** [wagtail/wagtail-localize#459] [Q&A]
I'm using wagtail_localize, and I want to disclude a CharBlock from being translated. In the wagtail_localize documentation, I only found the SynchronizedField, but there is no explanation on how to apply this to a CharBlock.

**Comment by enzedonline** [comment]
You can state which fields you want to be translatable. For example if you have a model that has a field called 'text' that you want to be the only field translated, you'd use the following:

translatable_fields = [
        TranslatableField('text'),
    ]

Just put the fields you want to translate in the translatable_fields list, leave out the ones you don't want translated.

**Reply by enzedonline** [reply]
Sorry, just realised that was a useless answer as it doesn't apply to StructBlocks ... would be good to have a solution to this.
